cfpb/consumerfinance.gov
# PfC: Replace keyCode with key

Paying for College has the last instance of using `keyCode`, which is deprecated. This PR replaces it with `key`.

See https://[GHE]/CFGOV/platform/issues/4391

## Changes

- PfC: Migrate deprecated keyCode to KeyboardEvent.key 


## How to test this PR

1. Visit http://localhost:8000/paying-for-college/your-financial-path-to-graduation/ and click through the application to the end and see that you can tab to the "Copy" button and hit the enter key to change it to "Copied" to copy the URL.


## Screenshots
<img width="150" alt="Screen Shot 2023-06-22 at 9 44 19 AM" src="https://github.com/cfpb/consumerfinance.gov/assets/704760/61bdbac0-7ad9-4ac0-beb1-bdfce33f79d7">
<img width="179" alt="Screen Shot 2023-06-22 at 9 44 23 AM" src="https://github.com/cfpb/consumerfinance.gov/assets/704760/5e6eb711-f4ba-421e-80a5-bf06d7f317b7">
